### DISCUSSION TITLE
How to upgrade to Transmission 4 on Ubuntu?

### DISCUSSION THREAD

**Discussion by zobjo** [transmission/transmission#4794] [Support]
Hi,
Sorry for the inconvenience but I'm a complete beginner in Linux (Ubuntu 22.10) and I don't know how to update my Transmission 3 to the new version 4 (I didn't find a .deb file) to upgrade easily. Can you help me please?

thanks

**Comment by fredo-47** [comment]
1. Download the new 4.0.0 transmission from here:
https://github.com/transmission/transmission/releases/download/4.0.0/transmission-4.0.0.tar.xz

2. Follow the step given here, while replacing the filename for 3.00 with the filename for 4.0.0
https://github.com/transmission/transmission#building-a-transmission-release-from-the-command-line
